language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java

```diff
@@ -79,6 +82,18 @@ public Optional<TypeSymbol> transform(LetVariableDeclarationNode letVariableDecl
         return getTypeDescriptorOfVariable(letVariableDeclarationNode);
     }
 
+    @Override
+    public Optional<TypeSymbol> transform(ObjectFieldNode objectFieldNode) {
+        return objectFieldNode.typeName().apply(this);
+    }
+
+    @Override
+    public Optional<TypeSymbol> transform(BuiltinSimpleNameReferenceNode builtinSimpleNameReferenceNode) {
```

Comment:
Can't we use SimpleNameReferenceNode instead? And could you provide an example which is covered by this transform method

Reply:
```
class A {

float f1 = 10.2;
decimal d1 = 1.2;

float f2 = f1 <cursor> / d1;

}
```
Above scenario is covered by this case.